### PR TITLE
Refine `Impl::Init` class and associates

### PR DESCRIPTION
**Pull request by ashmaroli**
- mark some methods in `Impl::Init` as `private`
- abstract repeating steps to another helper method
- fix a couple of typos and formatting
- slightly improve `help` display for `wabur` usage

### CODE REVIEW

File: lib/wab/impl/init.rb [hunk 1]
```diff
@@ -47,71 +48,56 @@ def exist_check(path)
...
-                                   {
-                                     kind: '#{type}',
-                                   }, ['$ref']))
-|
-            }
-            f.write(template % { rest_flows: rest_flows })
+          rest_flows = ''
+          types.each { |type|
+            rest_flows << %|
+    add_flow(UI::RestFlow.new(shell, { kind: '#{type}'}, ['$ref']))|
```
Comment (ohler55) [inline lib/wab/impl/init.rb]: It is better to have an expanded version so users can just add a line for every attribute. With the change they will have to reformat as immediately.

**Comment by ohler55** [comment]
I know there are no unit tests for this yet but have you tried running `wabur init` with the changes? Did it work correctly?

**Comment by ashmaroli** [comment]
> Did it work correctly?

Yes, it worked correctly..
Say, I were to include a unit-test for this.. what all do we check for..?

**Comment by ohler55** [comment]
I suppose the test would run a command with some number of types and then verify all the files were created and had the correct content.

**Comment by ashmaroli** [comment]
yep.. I figured so too.. will push a couple of basic assertions shortly

**Comment by ohler55** [comment]
oops, I thought it was ready. A new PR

**Comment by ohler55** [comment]
I noticed our editors seem to indent differently.

**Comment by ashmaroli** [comment]
> I noticed our editors seem to indent differently.

meaning..? Its probably not the editors.. I use Sublime Text too.. :smiley:

**Comment by ohler55** [comment]
I use emacs.
yours:
```
    add_flow(
      WAB::UI::RestFlow.new(
        shell,
        {
          kind: 'Bar',
        },
        ['$ref']
      )
    )
```
emacs
```
    add_flow(
             WAB::UI::RestFlow.new(
                                   shell,
                                   {
                                     kind: 'Foo',
                                   },
                                   ['$ref']
                                   )
    )
```
 Thats why my original was 
```
    add_flow(WAB::UI::RestFlow.new(shell,
                                   {
                                     kind: 'Foo',
                                   }, ['$ref']))
```
to also not have as many lines with just one character on them.

**Comment by ashmaroli** [comment]
mine was a deliberate move.. is it hard to read.?

**Comment by ohler55** [comment]
To me it seems a little off but that due to my eyes being used to the emacs formatting style. It also seems a little spread out. I think the first thing that will happen is the editor the developer is using will re-indent.

**Comment by ohler55** [comment]
What does the Ruby style guide suggest?

**Comment by ashmaroli** [comment]
The Ruby style-guide is fine with either of the above as long as the parameters are aligned properly..

**Comment by ohler55** [comment]
It goes against my preference and to me it is not clean but in the end it's not that important to me.

**Comment by ashmaroli** [comment]
Author preference trumps collaborator preference..
Feel free to update `develop` directly..

**Comment by ohler55** [comment]
I might unless you have strong feelings about it.

**Comment by ashmaroli** [comment]
Its fine.. change if you want to.. let the users re-format to their liking..

**Pull request merged**